packages/component-base/src/gestures.js

```diff
@@ -8,6 +8,8 @@ Code distributed by Google as part of the polymer project is also
 subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
 */
 
+/* eslint-disable @typescript-eslint/no-use-before-define */
```

Comment:
I didn't manage to sort methods in this file for a reasonable time. I think it is alright to disable the rule for this particular file as it is mostly copy-paste from Polymer.